src/connection/database/actions.js

```diff
@@ -322,11 +322,11 @@ export function showLoginActivity(id, fields = ['password']) {
 
 export function showSignUpActivity(id, fields = ['password']) {
   const m = read(getEntity, 'lock', id);
-  const captchaConfig = l.captcha(m);
+  const captchaConfig = l.signupCaptcha(m);
   if (captchaConfig && captchaConfig.get('provider') === 'arkose') {
     swap(updateEntity, 'lock', id, setScreen, 'signUp', fields);
   } else {
-    swapCaptcha(id, 'login', false, () => {
+    swapCaptcha(id, Flow.SIGNUP, false, () => {
       swap(updateEntity, 'lock', id, setScreen, 'signUp', fields);
```

Comment:
Camelcased 'signUp' must be used for this function which is why I couldn't use Flow.SIGNUP (which equals 'signup')

Reply:
Is there an opportunity to refactor this function to use lowercase "signup" so we have some uniformity?

Reply:
Unfortunately no. I looked into this and this is coming from auth0.js.